### ISSUE TITLE
Cursor not activating search area after pushing f

### ISSUE THREAD

**Issue by Ajayz12** [philc/vimium#2296]
Hi im a vimium user for maybe just 6 months now. When I push f on almost any website, then try to select the  search bar the cursor does not show up so i cant type anything in the search bar. I can get around this problem by creating a new tab and switching to that tab and then changing back to the original tab i was on and then I can use vimium by pushing f and then the search bar I select is activated and I am able to type in there. but why cant I just select the search bar the first time like before. Something has changed and I'm not sure what is the problem. Please help. AJ

**Comment by smblott-github** [comment]
Please provide an example page where this happens (and the sequence of steps necessary to reproduce the issue).

**Comment by Ajayz12** [comment]
https://www.youtube.com/

**Comment by smblott-github** [comment]
Thanks, @Ajayz12.  I can't seem to reproduce that behaviour, but I'll look out for it.

One possible explanation would be that there are multiple frames and, although the input is selected, the focus is in the wrong frame.

Do you have the *Don't let pages steal the focus on load* selected?  That could be at play too?

Does `gi` work?

**Comment by smblott-github** [comment]
@Ajayz12... I have a theory as to what might be going on here.

- When `f`, doesn't work, does `gi` work?
- Do you have the *Don't let pages steal the focus on load* enabled?

If the answers are "no" and "yes", then I think I know the problem.

Could you confirm (or otherwise), please?

**Comment by Ajayz12** [comment]
Hi there, I havent made any changes yet after reading your last comment. I'm just  confirming that "no" f still doesnt work and yes gi works

**Comment by Ajayz12** [comment]
Answer: So I went into chrome://extensions/ and noticed 2 check boxes in vimiums extension were unchecked. One box was the " Allow in cognito " and the other box was " Allow access to file URLs ".
After checking both boxes, I went back to https://www.youtube.com/ and the problem with selecting the search bar using 'f' was fixed. It was not a vimium fault just an had to make an allowance for vimium to have access to chrome some features. 

As I have just recently updated my computer, I have chrome v 53... so many changes were made without my knowledge. Thanks for trying to help in this matter. I am still very happy with this Vimium App. It has made surfing the net much faster and enjoyable.

Cheers AJ

![image](https://cloud.githubusercontent.com/assets/22624784/19249804/cd250e62-8f93-11e6-9615-2e9b07e0b65b.png)
